packages/cli/src/commands/deploy/index.ts

```diff
@@ -71,6 +70,7 @@ import { pickOverrides } from '../../util/projects/project-settings';
 import { printDeploymentStatus } from '../../util/deploy/print-deployment-status';
 import { help } from '../help';
 import { deployCommand } from './command';
+import { getSupportedNodeVersion } from '@vercel/build-utils/dist/fs/node-version';
```

Comment:
We should avoid importing for `dist` dirs.

```suggestion
import { getSupportedNodeVersion } from '@vercel/build-utils';
```